src/OnboardingSPA/components/MiniPreview/stylesheet.scss

```diff
@@ -1,230 +1,228 @@
-$white: #FFFFFF;
-$black: #000000;
-$browser-dark: #CACACA;
-$browser-light: #F2F2F2;
+$browser-dark: #cacaca;
+$browser-light: var(--nfd-onboarding-light-gray-3);
```

Comment:
can the brand variable be used directly instead of `$browser-light`, like how it's done in other files?

Reply:
done .. using the brand variable now